cmd/admin-handlers.go

```diff
@@ -501,7 +502,7 @@ func (adminAPI adminAPIHandlers) HealObjectHandler(w http.ResponseWriter, r *htt
 	writeSuccessResponseHeadersOnly(w)
 }
 
-// HealFormatHandler - POST /?heal
+// HealFormatHandler - POST /?heal&dry-run=[yes|no]
```

Comment:
Was thinking that we can avoid making "yes" or no for this. But instead have queryParam itself optional i.e without any value.

This means to heal
```
POST /?heal
``` 

This means fake heal
```
POST /?heal&dry-run
```

Thoughts?